envisage/tests/test_api.py

```diff
@@ -0,0 +1,27 @@
+# (C) Copyright 2007-2023 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+import unittest
+import envisage.api as api
+import envisage.ids as ids
+
+
+class ApiTestCase(unittest.TestCase):
```

Comment:
I suggest `TestApi`, to match the name of the module. (We do have a few existing `*TestCase` names, but in new code we should migrate away from that pattern.)

```suggestion
class TestApi(unittest.TestCase):
```